### PR TITLE
chore(pace-92): add welcome message

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: cmd/pace/main.go [hunk 1]
```diff
@@ -77,3 +80,46 @@ func CreateConfigDirAndFileIfNotExists() {
...
+	lastSeenCommandFilepath := path.Join(common.ConfigPath(), common.DefaultLastSeenFilename)
+	now := time.Now()
+	if content, err := os.ReadFile(lastSeenCommandFilepath); err != nil {
+		updateLastSeen(lastSeenCommandFilepath, now)
+	} else {
+		ts, err := strconv.ParseInt(strings.Trim(string(content), "\n"), 10, 64)
+		if err != nil {
+			os.Remove(lastSeenCommandFilepath)
+			updateLastSeen(lastSeenCommandFilepath, now)
+		}
```
Comment (ivan-p92) [inline cmd/pace/main.go:94]: When would this occur?

File: cmd/pace/main.go [hunk 2]
```diff
@@ -77,3 +80,46 @@ func CreateConfigDirAndFileIfNotExists() {
...
+---------------------------------------------------------------------
+                        ____   _    ____ _____                         
+                       |  _ \ / \  / ___| ____|
+                       | |_) / _ \| |   |  _|  
+                       |  __/ ___ \ |___| |___ 
+                       |_| /_/   \_\____|_____|
+---------------------------------------------------------------------
+Hey there, cool you're using PACE!
+We'd love to learn what your use case is or if you have any feedback.
+Join our Slack: https://getstrm.com/slack
```
Comment (ivan-p92) [inline cmd/pace/main.go:122]: Oh nice, is this url new?

**Review by ivan-p92 — APPROVED** [review]

File: cmd/pace/main.go [hunk 3]
```diff
@@ -77,3 +80,46 @@ func CreateConfigDirAndFileIfNotExists() {
 		))
 	}
 }
+
+func CreateLastSeenCommand() {
```
Comment (trietsch) [inline cmd/pace/main.go:84]: It's not really a command, is it?

File: cmd/pace/main.go [hunk 4]
```diff
@@ -77,3 +80,46 @@ func CreateConfigDirAndFileIfNotExists() {
...
+func printWelcomeMessage() {
+	asciiArt := `
+---------------------------------------------------------------------
+                        ____   _    ____ _____                         
+                       |  _ \ / \  / ___| ____|
+                       | |_) / _ \| |   |  _|  
+                       |  __/ ___ \ |___| |___ 
+                       |_| /_/   \_\____|_____|
+---------------------------------------------------------------------
+Hey there, cool you're using PACE!
```
Comment (trietsch) [inline cmd/pace/main.go:120]: Not sure whether a full blown ASCII art is the way to go. Wouldn't this be annoying every 24 hours? We could make an opt out with a setting?

File: cmd/pace/main.go (hunk 1 above)
Comment (bobvandenhoogen) [reply cmd/pace/main.go:94]: if the file is somehow corrupt, if it is incorrectly changed by user for example.

File: cmd/pace/main.go (hunk 2 above)
Comment (bobvandenhoogen) [reply cmd/pace/main.go:122]: yes it is!

File: cmd/pace/main.go (hunk 1 above)
Comment (ivan-p92) [reply cmd/pace/main.go:94]: Ah, clear, thx 👍

File: cmd/pace/main.go (hunk 3 above)
Comment (bobvandenhoogen) [reply cmd/pace/main.go:84]: sure

**Review by trietsch — APPROVED** [review]
Feel free to merge, but we should instruct users how to opt out.

**Review by trietsch — APPROVED** [review]

**Pull request merged**